tailwind-showcases/src/TailwindShowcases.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import 'twind/shim';
```

Comment:
I thought Georges said it's not ok here.

Reply:
Done @gqio